core/src/main/java/org/jobrunr/utils/reflection/ReflectionUtils.java

```diff
@@ -273,10 +263,11 @@ public static void makeAccessible(AccessibleObject accessibleObject) {
     }
 
     private static <T> Constructor<T> getConstructorForArgs(Class<T> clazz, Class<?>[] args) {
-        Constructor<?>[] constructors = clazz.getConstructors();
-
+        Constructor<?>[] constructors = clazz.getDeclaredConstructors();
```

Comment:
What's the reason you use `getDeclaredConstructors`?

Reply:
The lambda constructor is not public :(